web/src/components/EvidenceCard.tsx

```diff
@@ -96,6 +97,10 @@ const DesktopText = styled.span`
   )}
 `;
 
+const Timestamp = styled.p`
+  color: #BECCE5;
```

Comment:
instead of this:

`color: #BECCE5;`

do this: 

`color: ${({ theme }) => theme.secondaryText};`

this is because we have all the color scheme centralized for both dark mode and light mode, you can check on the `styles/themes.ts` folder